modules/serverless-blue-green-deploy.adoc

```diff
@@ -0,0 +1,112 @@
+[id="serverless-blue-green-deploy_{context}"]
+= Routing and managing traffic by using a blue-green deployment strategy
+
+You can safely reroute traffic from a live version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
+
+.Procedure
+
+. Create and deploy an app as a Knative service.
+. Find the name of the first revision that was created when you deployed the service, by viewing the output from the following command:
++
+[source,terminal]
+----
+$ oc get configurations <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
+. Add the following YAML to the service `spec` to send inbound traffic to the revision:
++
+.Example Route
+[source,yaml]
+----
+...
+spec:
+  traffic:
+    - revisionName: <first_revision_name>
+      percent: 100 # All traffic goes to this revision
+...
+----
+. Verify that you can view your app at the URL output you get from running the following command:
++
+[source,terminal]
+----
+$ oc get route <route_name>
+----
+. Deploy a second revision of your app by modifying at least one field in the `template` spec of the service. For example, you can modify the `image` of the service, or an `env` environment variable.
+. Redeploy the service by applying the updated `Service` resource. You can do this by applying the service YAML file, or by using the `kn service update` command if you have installed the `kn` CLI.
+. Find the name of the second, latest revision that was created when you redeployed the service, by running the command:
++
+[source,terminal]
+----
+$ oc get configurations <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
++
+At this point, both the first and second revisions of the service are deployed and running.
+. Update your existing route to create a new, test endpoint for the second revision, while still sending all other traffic to the first revision:
++
+.Example of updated route
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Route
+metadata:
+  name: <route_name>
+  namespace: default
```

Comment:
Should we drop this too? Also update the heading to say `updated service`?

Reply:
updated this a bit to make it consistent with earlier example